tests/components/overseerr/test_sensor.py

```diff
@@ -0,0 +1,33 @@
+"""Test the Overseerr sensor platform."""
+import pytest
+from syrupy.assertion import SnapshotAssertion
+from syrupy.filters import props
+
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry as er
+
+
+@pytest.mark.usefixtures("setup_integration")
+@pytest.mark.parametrize(
+    "sensor_name",
+    [
+        "movie_requests",
+        "tv_requests",
+        "approved_requests",
+        "available_requests",
+        "pending_requests",
+        "total_requests",
+    ],
+)
+async def test_overseerr_sensor(
+    hass: HomeAssistant,
+    entity_registry: er.EntityRegistry,
+    snapshot: SnapshotAssertion,
+    sensor_name: str,
+) -> None:
+    """Test the overseerr sensor states."""
+    entry = entity_registry.async_get(f"sensor.mock_title_{sensor_name}")
+    assert entry == snapshot(exclude=props("unique_id"))
```

Comment:
why are we excluding the `unique_id` here?

Reply:
When creating an entry, it is given a name based on its unique_id, which is added to the snapshot. On the next run, it creates another unique_id, which will never match the snapshot. This is why the id is excluded, open to suggestions though

Reply:
are you sure? You're fixing the `unique_id` in the `fixture_config_entry` so it should always be `12345`

Reply:
Yes, it creates an entry like this:
`9734ue324huj43_requested_approved`
I do not know why unfortunately